app/models/user.rb

```diff
@@ -78,30 +78,6 @@ def all_hooks
     all
   end
 
-  def payload
-    attrs = { "id" => id, "handle" => handle }
-    attrs["email"] = email unless hide_email
-    attrs
-  end
-
-  def as_json(*)
-    payload
-  end
-
-  def to_xml(options = {})
-    payload.to_xml(options.merge(root: 'user'))
-  end
-
-  def to_yaml(*args)
-    payload.to_yaml(*args)
-  end
```

Comment:
This code seems relatively simpler compared to the changes to the yaml renderer, controller, and new serializers extended to support yaml and xml.

I'm an AMS maintainer. I like AMS. It just doesn't seem an obvious improvement here, and my guess is there's also a performance hit.